ssi-jws/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 pub mod error;
 pub use error::Error;
+use k256::ecdsa::signature::Signature;
```

Comment:
All those imports are in the match statements. It shouldn't be needed, or if it is it should be in the correct block (so that it's feature-gated, I'm actually surprised the CI passed)